### PR TITLE
Add init command and beginning of build (parse settings.toml)

### PR DESCRIPTION
**Pull request by Keats**

### CODE REVIEW

**Comment by Keats** [comment]
I think passing around Option and exiting with a non-0 error code would be better than all those panic!, what do you think @deane ?

**Comment by Keats** [comment]
Updated to use Result, much shorter code

File: Cargo.toml [hunk 1]
```diff
@@ -5,3 +5,5 @@ authors = ["Vincent Prouillet <vincent@wearewizards.io>"]
 
 [dependencies]
 clap = "*"
+toml = "*"
+rustc-serialize = "*"
```
Comment (deane) [inline Cargo.toml]: are those libs stable enough to use unpinned versions?

Comment (Keats) [inline Cargo.toml]: it's ok, we want to track the latest versions of those, at least until it's stable

**Comment by Keats** [comment]
Turns out there is a default trait ! i'll implement that for the settings

**Pull request merged**